docs/source/user-manual/links-to-item.md

```diff
@@ -1,37 +1,37 @@
 ---
 myst:
   html_meta:
-    "description": "User manual on how to find all links to the current item."
-    "property=og:description": "User manual on how to find all links to the current item."
-    "property=og:title": "Finding links to the current item."
-    "keywords": "Volto, Plone, frontend, React, User manual, links"
+    "description": "User manual on how to find all links and relations to the current item."
+    "property=og:description": "User manual on how to find all links and relations to the current item."
+    "property=og:title": "Finding links and relations to the current item."
+    "keywords": "Volto, Plone, frontend, React, User manual, links, relations, references, related content"
 ---
 
 (links-to-item-label)=
 
-# Finding links to the current item
+# Finding links and references to the current page
 
 Sometimes it can be hard to keep track from where a certain page is referenced.
 This is especially true whenever you want to reconstruct your website with a greater number of objects being moved or deleted.
-For this situation, you can visit the {guilabel}`Links to item` page, which is an overview of all objects that refererence a certain object.
+For this situation, you can visit the {guilabel}`Links and references` page, which is an overview of all content items that refererence a certain content item.
 
 ```{note}
-An item references another item by creating a hyperlink to this item, using the item in a block, or using it in the {guilabel}`Related Items` field.
+An item references another item by creating a hyperlink to this item, using the item in a block, referencing it in the {guilabel}`Related Items` field, or referencing it in a relation field.
 ```
 
-## {guilabel}`Links to item` view
+## {guilabel}`Links and references` view
 
 In the toolbar on the left side, you can click on the item with the three horizontally aligned dots, called {guilabel}`More menu`.
-Click then on the item labeled {guilabel}`Links to item`.
+Click then on the item labeled {guilabel}`Links and references`.
 
-You can see now a table displaying all links to the current item.
+You can see now a table displaying all links and references to the current item.
 
 ```{image} ../_static/user-manual/manage/link-to-items.png
-:alt: A panel captioned with "Links to My Summer Vacation - Day 2". A table with two columns (first column labeled with "linked by this item", second column labeled with "review state"). In the row appears a link titled "Links to My Summer Vacation - Day 1", because it is referencing the current item.
+:alt: A panel captioned with "Links and references to 'My Summer Vacation - Day 2'". A table with three columns (first column labeled with "Linking / referencing this item…", second column labeled with "Review state", third column labeled with "Type"). In the row appears a link titled "Links to My Summer Vacation - Day 1", because it is referencing the current item.
```

Comment:
```suggestion
:alt: A panel captioned with "Content that links to or references 'Supervisor Adélaïde Pickavance'". Below the caption, there is a table with three sections, where each section has three columns. The first column's heading is the section name, the second is "Review State", and the third is "Type". The three sections are named, from top to bottom, "Linking this item with hyperlink in text", "Referencing the item as related item", and "Referencing this item with '[Name of Relation Field]'".
```

Reply:
If it is too difficult to describe in words, maybe the table is too complex? Maybe it would be easier to have three tables, instead of three sections in one table? Then we don't have do alt text gymnastics.

I don't know whether mentioning the table is generative would be helpful, and would cause further confusion.

Anyway, when I get stuck on what to do, I usually go with the simpler method. `¯\_(ツ)_/¯`